nixos/modules/services/web-apps/plausible.nix

```diff
@@ -11,13 +11,6 @@ in {
 
     package = mkPackageOptionMD pkgs "plausible" { };
 
-    releaseCookiePath = mkOption {
```

Comment:
Should we add `mkRemovedOptionModule` for this option?

Reply:
Good idea, done.